### DISCUSSION TITLE
local sound

### DISCUSSION THREAD

**Discussion by seancmalloy** [music-assistant/support#1660] [Q&A] [answered]
will music assistant ever be able to send audio to local sound card. this wasnt issue until now, i used to send audio to networked speakers  linked with a shairport-sync speaker for ledfx but recently started using addon which requires local sound. im aware that this topic was brought up in past just wondering the status.

**Comment by OzGav** [comment, marked as answer]
How do I stream to my local HA device  https://github.com/orgs/music-assistant/discussions/710

**Reply by OzGav** [reply]
Has this been answered? You can also stream to web browsers now with the Snapcast provider.

**Reply by seancmalloy** [reply]
Ill check that out. You can close. Thanks
